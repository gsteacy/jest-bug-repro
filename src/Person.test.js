const Person = require("./Person");

describe("tests", () => {
    test("defaults", () => {
        expect(Person()).toMatchSnapshot();
    });

    test("with name", () => {
        expect(Person("Jane Doe")).toMatchSnapshot();
    });

    // test("with age", () => {
    //     expect(Person(undefined, 78)).toMatchSnapshot();
    // });
});