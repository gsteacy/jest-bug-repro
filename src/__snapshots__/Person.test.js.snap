// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests defaults 1`] = `
Object {
  "age": 45,
  "name": "John Smith",
}
`;

exports[`tests with name 1`] = `
Object {
  "age": 45,
  "name": "Jane Doe",
}
`;
